lms/djangoapps/student_account/views.py

```diff
@@ -3,28 +3,35 @@
 import logging
 import json
 import urlparse
+import pytz
```

Comment:
Nitpick: this is a third-party library and should be in the second block of imports.